src/main/java/com/appdirect/sdk/vendorFields/controller/VendorRequiredFieldController.java

```diff
@@ -71,7 +75,8 @@
             @RequestParam(required = false, value = "salesAgentUserId") final String salesAgentUserId,
             @RequestParam(required = false, value = "salesAgentCompanyId") final String salesAgentCompanyId,
             @RequestHeader(required = false, value = "Accept-Language") final Locale locale,
-            @RequestHeader(value = "AD-Tenant") final String partnerCode) {
+            @RequestHeader(value = "AD-Tenant") final String partnerCode,
+            @RequestHeader() Map<String, String> headers) {
```

Comment:
can the parentheses be removed?
```suggestion
            @RequestHeader Map<String, String> headers) {
```